ocf_data_sampler/torch_datasets/datasets/site.py

```diff
@@ -62,7 +83,7 @@ def __init__(
         self.config = config
 
         # get all locations
```

Comment:
```suggestion
        # Get all locations
```

I know this wasn't introduced here, but I spotted it